cmd/preflight/cli/root.go

```diff
@@ -75,9 +79,20 @@ that a cluster meets the requirements to run an application.`,
 }
 
 func InitAndExecute() {
-	if err := RootCmd().Execute(); err != nil {
+	cmd := RootCmd()
+	err := cmd.Execute()
+
+	if err != nil {
+		var exitErr types.ExitError
+		if errors.As(err, &exitErr) {
+			if len(exitErr.Error()) > 0 {
+				cmd.PrintErrln("Error:", err.Error())
+			}
+			os.Exit(exitErr.ExitStatus())
+		}
 		os.Exit(1)
 	}
```

Comment:
you will need to print the error regardless as you are silencing errors in the cmd

```suggestion
	if err != nil {
		cmd.PrintErrln("Error:", err.Error())
		var exitErr types.ExitError
		if errors.As(err, &exitErr) {
			os.Exit(exitErr.ExitStatus())
		}
		os.Exit(1)
	}
```